air/src/aux_table/mod.rs

```diff
@@ -72,6 +75,10 @@ pub fn get_transition_constraint_count() -> usize {
         + memory::get_transition_constraint_count()
 }
 
+pub fn get_assertions(result: &mut Vec<Assertion<Felt>>) {
+    hasher::get_assertions_first_step(result);
+}
+
```

Comment:
I would prefer to use `get_assertions_first_step` here instead of `get_assertions` so that all of the internal modules follow the same pattern and the only `get_assertions` call is at the top level (in the AIR lib). I think this will keep things more consistent if we need to add any "last step" assertions within the aux table in the future.